mirdata/datasets/classicalDB.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+"""giantsteps_key Dataset Loader
+
+The ClassicalDB Dataset includes 880 classical music pieces across different styles from s.XVII to s. XX , annotated with
+single-key labels.
+
+Gómez, E. (2006). PhD Thesis. Tonal description of music audio signals.
+Department of Information and Communication Technologies.
+
+This dataset is mainly intended to assess the performance of computational key estimation algorithms in classical music.
+
+"""
+
+import json
+import librosa
+import os
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import core
+from mirdata import utils
+
+BIBTEX = """@article{gomez2006tonal,
+  title={Tonal description of music audio signals},
+  author={G{\'o}mez, Emilia},
+  journal={Department of Information and Communication Technologies},
+  year={2006}
+}"""
+# REMOTES = {
+#     []
+# }
+DOWNLOAD_INFO = """
+    Unfortunately the audio files of the classicalDB dataset are not available
+    for download. If you have the classicalDB audio dataset, place the contents into
+    a folder called GiantSteps_tempo with the following structure:
+        > classicalDB/
+            > audio/
+            > keys/
+    and copy the folder to {data_home}
```

Comment:
should we keep {data_home}? maybe they can load the dataset from its location

Reply:
Yes this issue is already fixed, @PRamoneda you need to remove `data_home` and just leave the brackets `{}`.

Reply:
mmmm ok

Reply:
done in a89861e